### ISSUE TITLE
Cable missing

### ISSUE THREAD

**Issue by Samuell95** [AppliedEnergistics/Applied-Energistics-2#955]
I have issue with ae2. Cable missing at the edge of chunk
![2015-02-28_12 10 15](https://cloud.githubusercontent.com/assets/3110002/6425992/1207210a-bf43-11e4-91b9-fe66ed16466c.png)
![2015-02-28_12 10 03](https://cloud.githubusercontent.com/assets/3110002/6425993/120d9fbc-bf43-11e4-8c9d-510ff3511aba.png)

Version:  rv2-beta-8

**Comment by TheJulianJES** [comment]
Are you using EnderIO Me Conduit?

**Comment by Samuell95** [comment]
No.

**Comment by TheJulianJES** [comment]
Do you have a security terminal?

**Comment by Samuell95** [comment]
Yes i have security terminal and version is rv2-beta-8

**Comment by thatsIch** [comment]
Is it actually missing or just not rendering

**Comment by yueh** [comment]
They can drop as part of the security grid, they question is mostly why and how to reproduce it.
For example the player id not getting written to disk or something like that.

**Comment by thatsIch** [comment]
maybe its cause chunks get loaded async and one network has permission while the other has not, and thus they cant connect, cause they are 2 different networks at that time?

**Comment by yueh** [comment]
Should not be an issue, as they persist their player id. 
But then I think they use a delayed loading of their NBT data. So maybe some race conditions?

The next build will contain a better logging of security conflicts, if enabled.
It will dump the coordinates as well as the owner to the log. Maybe this gives some hints about it (like different ids)

**Comment by thatsIch** [comment]
There are probably other stuff, we need to erase first (like security issues), before we can narrow the issue down.

You should build more frequently anyways, just do not forgot to edit the message. Sounds pretty cryptic atm

**Comment by yueh** [comment]
This is probably also a duplicate of #92 (together with #912 and other issues)
The old issue never really mentioned it occuring on world loading, so I mostly considered it as the (intended) breaking, if another player places a cable next to a protected network.

**Comment by Flukethoughts** [comment]
this is happening heavily on my server.

**Comment by halvors** [comment]
Happends on my server as well. But not that much recently. Running rv2-beta-14

**Comment by thatsIch** [comment]
Closing this in favor of https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/1074
